clang/lib/CIR/CodeGen/CIRGenModule.cpp

```diff
@@ -2361,6 +2386,19 @@ void CIRGenModule::setFunctionAttributes(GlobalDecl GD, mlir::cir::FuncOp F,
                                          bool IsIncompleteFunction,
                                          bool IsThunk) {
   assert(!MissingFeatures::setFunctionAttributes());
+  auto D = GD.getDecl();
+  const VisibilityAttr *VA = D->getAttr<VisibilityAttr>();
+  CIR_VisibilityAttr cir_visibility;
+  if (VA) {
+    cir_visibility = CIR_VisibilityAttr::get(
+        builder.getContext(),
+        getCIRVisibilityKindFromClangVisibility(VA->getVisibility()));
+  } else {
+    cir_visibility =
+        CIR_VisibilityAttr::get(builder.getContext(), VisibilityKind::Default);
```

Comment:
Instead of the name prefixed with CIR in `CIR_VisibilityAttr`, the name should be plain `VisibilityAttr`, use the name space to distinguish if needed, example: `mlir::cir::VisibilityAttr`